modules/migration-kubernetes-objects.adoc

```diff
@@ -16,13 +23,13 @@ After migration, the `closed` parameter of the `MigPlan` CR is set to `true`. Yo
 
 You add Kubernetes objects to the `MigPlan` CR by using one of the following options:
 
-* Adding the Kubernetes objects to the `includedResources` section.
+* Adding the Kubernetes objects to the `includedResources` section. When this field is specified in the `MigPlan` CR, the plan takes a list of `group-kind` as input. Only resources present in the list are included in the migration.
```

Comment:
@RichardHoch 
What does 'this field' refer to?  'includedResources section'?

Reply:
yes. I made that clearer.

Reply:
@RichardHoch 
What did you change? `includedResources` is still called 'section' and in the next sentence referred to as 'this field'.